peachjam/static/stylesheets/components/_timeline.scss

```diff
@@ -0,0 +1,74 @@
+.vertical-timeline {
+  position: relative;
+  max-width: 1200px;
+  margin: 0;
+
+  &:after {
+    content: '';
+    position: absolute;
+    width: 6px;
+    background-color: $gray-200;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    margin-left: -3px;
+  }
+}
+
+/* Container around content */
+.vertical-timeline__item {
+  padding: 10px 40px;
+  position: relative;
+
+  &:before {
+    content: " ";
+    position: absolute;
+    top: 22px;
+    z-index: 1;
+    left: 30px;
+    width: 0;
+    height: 0;
+    border-top: 10px solid transparent;
+    border-bottom: 10px solid transparent;
+    border-right: 10px solid $gray-100;
+  }
+
+  &:after {
+    left: -13px;
+    content: '';
+    position: absolute;
+    width: 25px;
+    height: 25px;
+    background-color: white;
+    border: 4px solid $primary;
+    top: 15px;
+    border-radius: 50%;
+    z-index: 1;
+  }
+}
+
+/* The actual content */
+.vertical-timeline__item__content {
+  padding: 20px 30px;
```

Comment:
Since we are using cards for the timeline items, as greg mentioned before, we can remove all styling of  `.vertical-timeline__item__content`, `. timeline__item__content__header `